src/components/project/dto/create-project.dto.ts

```diff
@@ -33,6 +34,7 @@ export abstract class CreateProject {
     description: 'Other location IDs',
     nullable: true,
   })
+  @EachItemIsNotBlank()
```

Comment:
Our existing `IsId` decorator should be sufficient. 
```suggestion
  @IsId({ each: true })
```

Reply:
oops. yeah, got a little carried away with making a new decorator 🤪

Reply:
np I only remember the `each` option because I read about it before and wrote an error message for it. I had to look it up again. It works right?